jujugui/static/gui/src/app/assets/css/_button.scss

```diff
@@ -0,0 +1,60 @@
+@import './mixins';
+@import './variables';
+
+/// Button classes
+/// @group Buttons
+@mixin juju-gui-buttons {
```

Comment:
These styles were moved here from the shared button file (they're GUI specific).